config/jobs/kubernetes/sig-node/dra-canary.yaml

```diff
@@ -21,6 +21,7 @@ presubmits:
     decoration_config:
       timeout: 90m
     path_alias: k8s.io/kubernetes
+    extra_refs:
```

Comment:
Can we somehow avoid having empty `extra_refs` all over the place?

Reply:
See https://github.com/kubernetes/test-infra/pull/35161#discussion_r2216258842

The alternative is to have an if check that combines all the individual if checks, which then needs to be maintained.

I've changed it, feel free to merge if you like that better.